src/utils/urlPaths.ts

```diff
@@ -11,11 +11,13 @@ export enum UrlPathTemplate {
   BasicDataInstitutions = '/basic-data/institutions',
   BasicDataPersonRegister = '/basic-data/person-register',
   Editor = '/editor',
-  EditorCurators = '/editor/curators',
-  EditorDoi = '/editor/doi',
-  EditorInstitution = '/editor/institution',
-  EditorPublishStrategy = '/editor/publish-strategy',
-  EditorVocabulary = '/editor/vocabulary',
+  EditorCurators = '/editor/overview/curators',
+  EditorDoi = '/editor/settings/doi',
+  EditorInstitution = '/editor/settings/institution',
+  EditorOverview = 'editor/overview',
+  EditorPublishStrategy = '/editor/settings/publish-strategy',
+  EditorSettings = 'editor/settings',
```

Comment:
bevisst at disse ikke starter med `/`?
```suggestion
  EditorOverview = '/editor/overview',
  EditorPublishStrategy = '/editor/settings/publish-strategy',
  EditorSettings = '/editor/settings',
```

Reply:
om de starter med `/` kan vi kanskje bruke `startsWith` i stedet for `includes` igjen også?